### ISSUE TITLE
[Bug] Problems when upgrading to 1.21.1

### ISSUE THREAD

**Issue by ChrisTVH** [SerbanHiro/ToastedAFK#6]
# About the bug
- It happens that when I updated to 1.21.1, the plugin gave me an error related to not finding the region, it had happened to me before when I updated to 1.21, but when it gives error, it doesn't even work, I can't reload it or use it.
- I have to restart the server, delete the `region:` section from the configuration to make it work.
![image](https://github.com/user-attachments/assets/8a67bc7a-e3cc-4849-a719-2955c70bdf89)
- In short, there is no error when using the commands, and it does not even respond.
## Server information
- ToastedAFK ver 2.0.2.2
- Purpur 1.21.1 build 2300
